segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/ZKSegmentContainerMonitor.java

```diff
@@ -98,6 +102,9 @@
         this.hostContainerMapNode = new NodeCache(zkClient, clusterPath);
         this.assigmentTask = new AtomicReference<>();
         this.lastReportTime = new AtomicLong(CURRENT_TIME_MILLIS.get());
+        // Allow OrderedItemProcessor to throw exceptions, as they may happen while starting containers. But we want to
+        // continue processing subsequent container starts irrespective of such failures.
+        this.startContainerOrderedProcessor = new OrderedItemProcessor<>(1, this::startContainer, false, this.executor);
```

Comment:
I would feel much better if we made this number configurable. Let's try to do that.

Reply:
I added the possibility of configuring it.